packages/react-tabs/Spec.md

```diff
@@ -235,22 +253,27 @@ TabContextValue provides context from TabsList
 - single-selection behavior
 - click handling
 
-> TODO: How best to provide a default selectionIndicator slot from a parent?
+> TODO: To provide a selectionIndicator slot implementation from the parent
+> should it be IntrinsicShorthandProps or IntrinsicShorthandValue?
 
 ```ts
 export type TabContextValue = Partial<Omit<TabCommons, 'value'>> & {
+  appearance: 'transparent' | 'subtle';
   selected: boolean;
   selectionIndicator: IntrinsicShorthandProps<'div'>;
+  size: 'small' | 'medium';
+  vertical: boolean;
   onClick(ev: React.MouseEvent | React.KeyboardEvent): void;
```

Comment:
Can we remove this or is there any manipulation necessary to the `onClick` event?

Reply:
since we always allow native DOM attr spreading, we probably shouldn't include event handlers unless they do something that's not standard

Reply:
My assumption is that it's there for the selection management but if that's the case, all possible "clicking" events should be handled.